packages/dd-trace/src/appsec/reporter.js

```diff
@@ -101,16 +101,16 @@ function reportWafInit (wafVersion, rulesVersion, diagnosticsRules = {}) {
   incrementWafInitMetric(wafVersion, rulesVersion)
 }
 
-function reportMetrics (metrics, raspRuleType) {
+function reportMetrics (metrics, raspRule) {
```

Comment:
I think that we should change also tests related with raspRule to avoid confusion on the usage, the tests are implemented with string and not with an object.

Reply:
Done